tools/hmac/payload.json

```diff
@@ -0,0 +1,54 @@
+{
+  "live": "false",
+  "notificationItems": [
```

Comment:
Rename file to `authorisation_payments_webhook_payload.json`

Reply:
It is an example, it can be any event